src/seedu/addressbook/data/AddressBook.java

```diff
@@ -82,6 +82,19 @@ public void addPerson(Person toAdd) throws DuplicatePersonException {
         allPersons.add(toAdd);
     }
 
+    public void editPerson(ReadOnlyPerson target, Person editedPerson) throws PersonNotFoundException {
```

Comment:
Every public facing method should have a method header according to JavaDoc standards.